internal/controller/provisioner/objects.go

```diff
@@ -33,9 +33,21 @@ const (
 	defaultServiceType   = corev1.ServiceTypeLoadBalancer
 	defaultServicePolicy = corev1.ServiceExternalTrafficPolicyLocal
 
-	defaultNginxImagePath     = "ghcr.io/nginx/nginx-gateway-fabric/nginx"
-	defaultNginxPlusImagePath = "private-registry.nginx.com/nginx-gateway-fabric/nginx-plus"
-	defaultImagePullPolicy    = corev1.PullIfNotPresent
+	defaultNginxImagePath        = "ghcr.io/nginx/nginx-gateway-fabric/nginx"
+	defaultNginxPlusImagePath    = "private-registry.nginx.com/nginx-gateway-fabric/nginx-plus"
+	defaultNginxPlusWafImagePath = "private-registry.nginx.com/nginx-gateway-fabric/nginx-plus-waf"
+	defaultImagePullPolicy       = corev1.PullIfNotPresent
+
+	// WAF container defaults.
+	defaultWAFEnforcerImagePath  = "private-registry.nginx.com/nap/waf-enforcer"
+	defaultWAFConfigMgrImagePath = "private-registry.nginx.com/nap/waf-config-mgr"
+	// FIXME(ciarams87): Figure out best way to handle WAF image tags.
+	defaultWAFImageTag = "5.6.0"
```

Comment:
Is this version publicly available where we can use renovate to update it?

Reply:
I don't think so, as far as I know it's only published in the private registry